### DISCUSSION TITLE
Can't call js function with webView.CoreWebView2.ExecuteScriptAsync

### DISCUSSION THREAD

**Discussion by tiger2014** [MicrosoftEdge/WebView2Feedback#4006] [Q&A]
I have a simple html:
`<!DOCTYPE html>`
`<html lang="en" dir="ltr">`
`  <head>`
`    <meta charset="utf-8">`
`  </head>`
`  <body>`
`    <div id="tvchart"></div>`
`  </body>`
`  <script src="https://unpkg.com/lightweight-charts/dist/lightweight-charts.standalone.production.js"></script>`
`  <script type="text/javascript" src="IndexWithMarkers6.js"></script>`
`  <script>`
`      function updateData(data) {`
`          console.log(data);`
`      }`
`  </script>`
`</html>`

I call the `updateData` in .NET: `await webView.CoreWebView2.ExecuteScriptAsync("updateData('yyy')");`, but the call is failed. How to do to call the function updateData?

**Comment by leonidukg** [comment]
On the test site, everything works correctly. 

```
namespace WpfApp1test
{
    /// <summary>
    /// Interaction logic for MainWindow.xaml
    /// </summary>
    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();
        }
        private async void Button_Click(object sender, RoutedEventArgs e)
        {
            await Xweb.CoreWebView2.ExecuteScriptAsync("updateData('yyy')");
        }
    }
}
```

![This is an image](https://i.paste.pics/18746a1ce29ab1ad5c54600737bd4708.png)

So the problem is solely in your code.
